hathor/event/model/event_data.py

```diff
@@ -52,8 +52,8 @@ class TxMetadata(BaseModel, extra=Extra.ignore):
     received_by: list[int]
     children: list[str]
     twins: list[str]
-    accumulated_weight: float
-    score: float
+    accumulated_weight: str
```

Comment:
Can this have impact on the wallet service and other use case using the reliable integrations? Should we calculate `log2(...)` to keep the model consistent? If yes, should we do it on the other APIs too? I mean, is there any benefit on using the whole integer instead of its more friendly log2 representation?

Reply:
I think the event data and the API data should be consistent, so I'll consider this for how we serialize the API data too. This would be the time to split the representation of the data for API and for the storage (currently it's the same method).

I do prefer to not change any existing field, so we would keep `accumulated_weight: float` and `score: float`.

However we could still expose the actual raw value somehow. Maybe we can do this on a different PR and explore the options there. For example we could add `accumulated_weight_raw: str` and `score_raw: str` if a `raw=true` parameter is given, although this would only work for APIs, event data wouldn't have a simple toggle.

Reply:
@msbrogli @glevco What do you think about keeping `accumulated_weight: float` and `score: float` in this PR by converting them from the internal `int` values using `work_to_weight` when serializing for APIs and event-data?

And we can discuss if it's worth to somehow expose the raw int values in a future PR.

Reply:
I agree to keep the event data and the API data consistent. I guess we can use this PR to add the raw values too as suggested. So use cases can pick whatever is best for them.

Reply:
You did it! Thanks!